Dockerfile

```diff
@@ -1,8 +1,12 @@
 FROM node:10 as node_build
 
 COPY package.json package-lock.json ./
-COPY npm_install_wrapper.sh  ./
-RUN ./npm_install_wrapper.sh
+# On the Git protocol side, unencrypted git:// offers no integrity or authentication, making it subject to tampering.
+# We expect very few people are still using this protocol, especially given that you can’t push (it’s read-only on GitHub).
+# We’ll be disabling support for this protocol.
+# https://github.blog/2021-09-01-improving-git-protocol-security-github/#no-more-unauthenticated-git
+RUN git config --global url."https://".insteadOf git://
```

Comment:
What's the justification for keeping this? I don't think there is a threat worth worrying about, to start monkeying with devs' global git config. This old protocol is not used in the circleci build or README for devs, so surely it just adds complication?

Reply:
The docker image doesn't build on circle without this - its just the Docker image, so it doesn't affect anyone's machines any more.

Reply:
Ah, I tried it: https://app.circleci.com/pipelines/github/ministryofjustice/fala/1317/workflows/de9b4bd4-93c4-4707-862b-a1e96c807adc/jobs/5429/parallel-runs/0/steps/0-107 and it times out during `npm install`

So the reason isn't (just) security, it's some install issue. I guess one of the old node packages tries to get something from github using `git://`. If this is the case, then wouldn't it happen on our local machines too? Have you tested deleting your node_packages and rerunning npm install?